pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -69,3 +73,43 @@ type HyperConvergedList struct {
 func init() {
 	SchemeBuilder.Register(&HyperConverged{}, &HyperConvergedList{})
 }
+
+type KubevirtConfigurations struct {
+	ResourceVersion                   string            `json:"ResourceVersion,omitempty"`
+	UseEmulation                      bool              `json:"UseEmulation,omitempty"`
```

Comment:
Let's clean things up a little bit. We should group developer options like `UseEmulation` in a sub-struct like e.g. `dev`.